src/alpine/3.15/helix/amd64/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN cd /tmp && \
     cd pwsh && \
     curl -L https://github.com/PowerShell/PowerShell/releases/download/v7.2.5/powershell-7.2.5-linux-alpine-x64.tar.gz | tar xfz - && \
     cd .. && \
-    git clone --depth 1 --single-branch --branch release/7.0 --recursive https://github.com/dotnet/msquic && \
+    git clone --depth 1 --single-branch --branch main --recursive https://github.com/dotnet/msquic && \
```

Comment:
Does this image contain `libnuma-dev`?

Reply:
yes, it does. I did not add it explicitly so it is probably pulled in via dependency
```
fd3c746d0f75:/$ apk list | grep numa
numactl-2.0.16-r1 x86_64 {numactl} (LGPL-2.1-only) [installed]
numactl-dev-2.0.16-r1 x86_64 {numactl} (LGPL-2.1-only) [installed]
numactl-doc-2.0.16-r1 x86_64 {numactl} (LGPL-2.1-only)
numactl-tools-2.0.16-r1 x86_64 {numactl} (GPL-2.0-only)
```

more importantly, msquic builds - I assume this is what you ask